src/crsAdapter/BewotecExpertAdapter.js

```diff
@@ -313,18 +280,77 @@ class BewotecExpertAdapter {
                     return axios.get(baseUrl, {params: params});
                 }
 
-                // will window.open be a better solution? but when shall we then close the window?
-
                 this.logger.warn('HTTPS detected - will use img.src for data transfer');
 
                 // will create a mixed content warning
+                // will window.open be a better solution? but when shall we then close the window?
                 (new Image()).src = baseUrl + '?' + querystring.stringify(params);
 
                 return Promise.resolve();
             },
         };
     }
 
+    /**
+     * @private
+     * @param options object
+     * @returns {*}
+     */
+    getDataFromBewotecBridge(options) {
+        return new Promise((resolve, reject) => {
+            const bewotecDataListener = (message) => {
```

Comment:
'(message)' can be without parentheses, right?

Reply:
could be, yes. but i like to see on the first sight, what is a function parameter and what only a variable.
for example - if you read that: `const myVar = message = ...` you can not be sure, what that is. it could result in `const myVar = message => message.data;` or `const myVar = message = data;`

so, i stick to the syntax with the paranthesis.